src/main/java/by/yahor_kulesh/config/ConnectionConfig.java

```diff
@@ -1,15 +1,23 @@
 package by.yahor_kulesh.config;
 
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.SQLException;
 
+
+@Component
 public class ConnectionConfig {
-    private static final String url = "jdbc:postgresql://localhost:5432/my_ticket_service_db";
-    private static final String user = "postgres";
-    private static final String password = "postgres";
+    @Value("${connectionConfig.url}")
+    private String url;
+    @Value("${connectionConfig.username}")
+    private String user;
+    @Value("${connectionConfig.password}")
+    private String password;
 
-    public static Connection getConnection() throws SQLException {
+    public Connection getConnection() throws SQLException {
         return DriverManager.getConnection(url, user, password);
     }
```

Comment:
instead of manual configuration of Connection and choosing Driver manager use DataSource specific db bean:
```java
    @Bean
    public DataSource dataSource(){
        PGSimpleDataSource dataSource = new PGSimpleDataSource();
        dataSource.setURL(...);
        dataSource.setUser(...);
        dataSource.setPassword(...);
        return dataSource;
    }
```

Reply:
done